modules/history/src/hooks/after_event_processed/appendIncomingMsg.js

```diff
@@ -0,0 +1,12 @@
+const appendIncomingMsg = async () => {
+  if (event.preview && event.type != 'visit') {
+    await bp.database('msg_history').insert({
+      created_on: event.createdOn,
+      thread_id: event.threadId,
+      bot_id: event.botId,
+      msg_content: JSON.stringify(event)
```

Comment:
same comment for JSON helper (won't comment everywhere but check for your other files)